.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         julia-version:
           - "1"
```

Comment:
1 and 1.7 at the moment are the same thing. It's probably better to have 1.6 (for the oldest supported version) and 1 (for the latest stable version)

Reply:
I keep forgetting whether 1 is stable or LTS!

Reply:
"1" is resolved to be the latest version in the 1.x series. At the moment there is no automatic way to say "give me the LTS": https://github.com/JuliaLang/VersionsJSONUtil.jl/issues/2